### DISCUSSION TITLE
Firewall exclusion

### DISCUSSION THREAD

**Discussion by nikoslintovois** [jlesage/docker-firefox#218] [Q&A] [answered]
I cant access the internet with firewall enabled. When firewall is disabled i can access the web normally. What exclusion i must do to allow me to access the web? thanks

**Comment by jlesage** [comment]
You should provide more details about your setup.  What firewall are you using, one what device, etc.

**Reply by nikoslintovois** [reply]
I have ds224+ and i have enabled ds's firewall.

**Reply by jlesage** [reply, marked as answer]
It seems to have several tutorials on how to configure firewall on Synology.  There is also information from Synology forum.  For example, these might help you:
https://mariushosting.com/synology-how-to-correctly-set-up-firewall-on-dsm-7/ (see rule 6)
https://community.synology.com/enu/forum/11/post/142221

**Comment by nikoslintovois** [comment]
Thanks i found the solution from the link above. I had to allow firefox's subnet too found on container manager - network
